systers_portal/templates/meetup/snippets/users-grid.html

```diff
@@ -0,0 +1,16 @@
+<div class="row mt40">
+  <h2 class="ml15"> {{ users }} </h2>
+  <div class="col-sm-3">
```

Comment:
The columns seem too wide for small devices. I tried with `col-xs-3` and it looks better.